lighthouse-core/audits/estimated-input-latency.js

```diff
@@ -18,11 +20,12 @@ class EstimatedInputLatency extends Audit {
     return {
       name: 'estimated-input-latency',
       description: 'Estimated Input Latency',
-      helpText: 'The score above is an estimate of how long your app takes to respond to user ' +
-          'input, in milliseconds. There is a 90% probability that a user encounters this amount ' +
-          'of latency, or less. 10% of the time a user can expect additional latency. If your ' +
-          'latency is higher than 50 ms, users may perceive your app as laggy. ' +
-          '[Learn more](https://developers.google.com/web/tools/lighthouse/audits/estimated-input-latency).',
+      helpText:
```

Comment:
possible to revert these? don't appear to have changed :)

Reply:
done